mysql: add 8.1 release series and refresh releases for 5.7 and 8.0 (#3289)
captn3m0/endoflife.date: products/mysql.md

releases:
-   releaseCycle: "8.1"
    latest: '8.1.0'
    support: true

>     eol: false

Author: 👍

> web-based applications, used by high profile web properties including Facebook, Twitter, YouTube,
> Yahoo! and many more.

Oracle offers two release streams for MySQL:
[**Innovation** and **LTS** releases](https://blogs.oracle.com/mysql/post/introducing-mysql-innovation-and-longterm-support-lts-versions).

Innovation releases are similar to MySQL 8.0 continuous development model (< 8.0.34) and include bugfixes,
security patches, and new features. Support for Innovation releases is limited until the next minor release
(Innovation or LTS). Patch releases within Innovation releases are possible but are less likely to happen.

> Innovation releases are considered GA are recommended to be used in production environments.

Reviewer: ```suggestion
Innovation releases are production-grade quality and can be safely used in production environments.
```
